public/app/plugins/panel/graph2/GraphPanel.tsx

```diff
@@ -10,7 +10,7 @@ import { Options } from './types';
 interface Props extends PanelProps<Options> {}
 
 export class GraphPanel extends PureComponent<Props> {
-  constructor(props) {
+  constructor(props: Props) {
```

Comment:
I think we can get rid of this constructor as React will set props anyway.

Reply:
💯 right!